device-data/types/bolus/dual-square.md

```diff
@@ -60,7 +60,7 @@ We plan to migrate all Tidepool data to use `combo` as the value of this sub-typ
 		platform: yes
 	Numerical type: Floating point value rounded to the appropriate significant figures for the device's precision.
 	Range:
-		min: > 0.0
+		min: >= 0.0
```

Comment:
This bit is auto-generated and shouldn't be edited in the Markdown. I've spent some time today documenting how to work with the docs: [here](http://developer.tidepool.io/data-model/) and [here](http://developer.tidepool.io/data-model/bin/index.html). LMK if those make sense.